doc/functions/enum.rst

```diff
@@ -0,0 +1,26 @@
+``enum_cases``
+==============
+
+.. versionadded:: 3.15
+
+    The ``enum`` function was added in Twig 3.15.
+
+``enum`` gives access to enums:
+
+.. code-block:: twig
+
+    {# display one specific case of a backed enum #}
+    {{ enum('App\\MyEnum').SomeCase.value }}
+
+    {# get all cases of an enum #}
+    {% enum('App\\MyEnum').cases() %}
```

Comment:
Should we deprecate `enum_cases()`?

Reply:
I would not deprecated it. `enum_cases` compiles to optimized code when the enum class name is a string literal, avoiding any dot operator resolution at runtime.

Reply:
👍